### ISSUE TITLE
Support 'allow merge' config on an org and project basis

### ISSUE THREAD

**Issue by zanerock** [liquid-labs/liq-cli#57]
## Overview

The default config (soft) blocks merges (use PRs instead). On an org and project basis, the org config should allow overrides. When set to true, then `liq work merge` no longer needs to be forced.

## Deliverables

Specific deliverables are waiting on development of an org configuration standard.

**Comment by zanerock** [comment]
With full support for public and private in new implementation, this is no longer needed.
